plugins/src/main/java/opengrok/auth/plugin/UserWhiteListPlugin.java

```diff
@@ -18,7 +18,7 @@
  */
 
 /*
- * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Aren't we using `2019, 2020,` format?

Reply:
yes, fixed. There should be a copyright format checker I guess. Filed #3300.